test-app/package-lock.json

```diff
@@ -12059,8 +12060,7 @@
         },
         "yallist": {
           "version": "3.0.3",
-          "bundled": true,
-          "optional": true
+          "bundled": true
         }
       }
     },
```

Comment:
- not sure why there should be any changes here in `test-app`

Reply:
@dannyscot I am not sure What triggered these changes. Should I revert back these?

Reply:
typically, we don't check in changes to `package-lock.json` unless there are `package.json` changes OR intentional `package-lock.json` changes

Reply:
We should add a `.nvmrc` file, I suspect the `package-lock.json` changes are just due to varying version of node/npm being using.